src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs

```diff
@@ -36,6 +35,7 @@ public ExtensionBundleManager(ExtensionBundleOptions options, IEnvironment envir
             _cdnUri = _environment.GetEnvironmentVariable(EnvironmentSettingNames.ExtensionBundleSourceUri) ?? ScriptConstants.ExtensionBundleDefaultSourceUri;
             _options = options ?? throw new ArgumentNullException(nameof(options));
             _configOption = configOption ?? throw new ArgumentNullException(nameof(configOption));
+            _platformReleaseChannel = _environment.GetEnvironmentVariable(EnvironmentSettingNames.AntaresPlatformReleaseChannel) ?? ScriptConstants.LatestPlatformChannelName;
```

Comment:
using "Latest" as the default - this matches with previous behavior before considering this setting.

Reply:
Or move `_cdnUri` down so its between `_configOption` and `_platformReleaseChannel`